src/amp.js

```diff
@@ -0,0 +1,4 @@
+import * as aphrodite from 'aphrodite/no-important';
```

Comment:
Although you plan to use this for AMP, it is not AMP specific. I think this file should be renamed to no-important.js.